packages/smart-account/src/paymaster-helper/index.ts

```diff
@@ -0,0 +1,31 @@
+import { PaymasterAPI } from '@biconomy/account-abstraction'
+import { BiconomyVerifyingPaymasterAPI } from '@biconomy/account-abstraction'
+import { BiconomyTokenPaymasterAPI } from '@biconomy/account-abstraction'
+
+export type SupportedGasToken = 'USDC' | 'TEST_ERC20'
+
+export async function getPaymaster(
```

Comment:
@arcticfloyd1984 @tomarsachin2271 @divyaN73 @talhamalik883 
like this example
in the entry point of sdk we somehow need a way to know which token paymaster instance should be created..